src/global-messages/converters.ts

```diff
@@ -59,6 +73,11 @@ function mapToContexts(data: any): GlobalMessageContext[] | undefined {
     .filter(Boolean) as GlobalMessageContext[];
 }
 
+function isAppPlatformValid(platforms: AppPlatformType[]) {
+  if (!platforms) return true;
+  return !!platforms.find((platform) => platform === Platform.OS);
```

Comment:
Wondering if we should make this comparison case insensitive. wdut?

Reply:
Yeah, seems like a good idea :+1: